docs/config.md

```diff
@@ -318,6 +319,100 @@ location /blog/ {
 
 Default: `[]`
 
+- `NoDNSLink`  
+  A boolean to configure whether DNSLink lookup for value in `Host`
+  HTTP header should be performed.  If DNSLink is present, content path stored
+  in the DNS TXT record is mounted at `/` and respective payload is returned to
+  the client.
+
+Default: `false`
+
+### Gateway.PublicGateways
+
+`PublicGateways` is a dictionary for defining gateway behavior on specific fully qualified domain names (FQDN).  
```

Comment:
Are we doing per-gateway dnslink settings?

Reply:
Yes. 
`NoDNSLink` is both global and per host.
It enables user to opt-out from resolving `Host: ` header globally, then enable it for a specific hostname.